homeassistant/components/environment_canada/camera.py

```diff
@@ -31,23 +35,31 @@
         vol.Inclusive(CONF_LATITUDE, "latlon"): cv.latitude,
         vol.Inclusive(CONF_LONGITUDE, "latlon"): cv.longitude,
         vol.Optional(CONF_PRECIP_TYPE): vol.In(["RAIN", "SNOW"]),
+        vol.Optional(CONF_RADIUS, default=200): cv.positive_int,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler